stock_picking_progress/hooks.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+from odoo.tools.sql import column_exists, create_column
+
+logger = logging.getLogger(__name__)
+
+
+def setup_move_line_progress(cr):
+    table = "stock_move_line"
+    column = "progress"
+    _type = "float"
+    if column_exists(cr, table, column):
+        logger.info(f"{column} already exists on {table}, skipping setup")
+        return
+    logger.info(f"creating {column} on table {table}")
+    create_column(cr, table, column, _type)
+    fill_column_query = """
+        UPDATE stock_move_line
+        SET progress = CASE
+            WHEN (product_uom_qty IS NULL or product_uom_qty = 0.0) THEN 0.0
+            ELSE (qty_done / product_uom_qty) * 100
+        END;
+    """
+    logger.info(f"filling up {column} on {table}")
```

Comment:
```suggestion
    logger.info("filling up %s on %s", column, table)
```
Let it do its job and strings will be interpolated when the log is produced - if needed - and not at run time.

Reply:
modified this, testing the hook locally before pushing the change.

Reply:
@simahawk updated